cpp/src/aztec3/circuits/rollup/base/native_base_rollup_circuit.cpp

```diff
@@ -196,9 +198,10 @@ std::array<NT::fr, 2> calculate_calldata_hash(BaseRollupInputs baseRollupInputs,
         // Nullifiers
         auto new_nullifiers = baseRollupInputs.kernel_data[i].public_inputs.end.new_nullifiers;
         auto new_commitments = baseRollupInputs.kernel_data[i].public_inputs.end.new_commitments;
-        for (size_t j = 0; j < 4; j++) { // TODO: const
-            calldata_hash_inputs[i * 4 + j] = new_nullifiers[j];
-            calldata_hash_inputs[8 + i * 4 + j] = new_commitments[j];
+        for (size_t j = 0; j < KERNEL_NEW_COMMITMENTS_LENGTH; j++) { // TODO: const
+            calldata_hash_inputs[i * KERNEL_NEW_COMMITMENTS_LENGTH + j] = new_nullifiers[j];
+            calldata_hash_inputs[(KERNEL_NEW_NULLIFIERS_LENGTH * 2) + i * KERNEL_NEW_NULLIFIERS_LENGTH + j] =
```

Comment:
can you add a comment to increase readability:
"`KERNEL_NEW_NULLIFIERS_LENGTH * 2` to offset for all nullifiers"
Mayble also add a var for `i * KERNEL_NEW_COMMITMENTS_LENGTH + j` - it is used in other methods too. Can call this nullifier_placeholder_index